packages/components/addon/components/hds/pagination-bar/pagination/nav-button.hbs

```diff
@@ -0,0 +1,27 @@
+<Hds::Interactive
+  class={{this.classNames}}
+  {{on "click" this.onClick}}
+  @onClick={{@onClick}}
+  @direction={{@direction}}
+  aria-label={{if this.isNext "Next page" "Previous page"}}
+  disabled={{@disabled}}
+  ...attributes
+>
+  {{#if (eq @direction "next")}}
+    {{#if (eq @type "compact")}}
```

Comment:
Makes sense, thanks. I will change it.

Reply:
I decided instead to just clarify the documentation since currently the NavButton component is only used internally and I didn't know if it made sense to add another parameter to the Pagination wrapper component. We can reexamine later on though.